db/migrate/20160113221031_add_confirmable_to_admin.rb

```diff
@@ -0,0 +1,19 @@
+class AddConfirmableToAdmin < ActiveRecord::Migration
+  def self.up
+    add_column :admins, :confirmation_token, :string
+    add_column :admins, :confirmed_at, :datetime
+    add_column :admins, :confirmation_sent_at, :datetime
+    add_column :admins, :unconfirmed_email, :string
+    add_index :admins, :confirmation_token, :unique => true
+
+    Admin.update_all({
+      :confirmed_at => DateTime.now,
+      :confirmation_sent_at => DateTime.now
+    })
+  end
+
+  def self.down
+    remove_column :admins,
+                  [:confirmed_at, :confirmation_token, :confirmation_sent_at]
```

Comment:
OK, should be fixed.  `confirmation_token` was already being removed, and its index (which you may have been referring to in your comment) should be removed when the column is dropped.